scripts/inner/kubestellar

```diff
@@ -38,54 +38,63 @@ remove=0
 cleanup=0
 espw_name="espw"
 log_folder=$(pwd)/kubestellar-logs
-local_kcp=true
 imws="root:imw1"
 wmws="root:wmw1"
 
 function echoerr() {
    echo "ERROR: $1" >&2
 }
 
+function run_kubectl_in_space() { # usage: space_config kubectl_cmd
+    space_config="$1"
+    cmd="$2"
+    KUBECONFIG=$space_config kubectl $cmd
+}
```

Comment:
This function adds little convenience and introduces the possibilities of bugs due to the shell processing the command line twice. Consider, for example, what happens if one of the arguments to `run_kubectl_in_space` has a space in it. I think it would be clearer to just not have this function, let the call sites just say themselves `KUBECONFIG=<whatever> kubectl ...`.

Reply:
function removed.